`out_stream.write()` (a.k.a. `write_raw`) raises `TypeError`

`out_stream.write()` raises `TypeError` for both library and gRPC codepaths:
```
FAILED tests/component/test_out_stream.py::test___analog_single_channel_task___write_raw___returns_samples_written[library_init_kwargs] - ctypes.ArgumentError: argument 2: <class 'TypeError'>: wrong type
FAILED tests/component/test_out_stream.py::test___analog_single_channel_task___write_raw___returns_samples_written[grpc_init_kwargs] - TypeError: 'float' object cannot be interpreted as an integer
```

The gRPC codepath gives a more helpful error:
```
        response = self._invoke(
            self._client.WriteRaw,
>           grpc_types.WriteRawRequest(
                task=task, num_samps=num_samps, auto_start=auto_start,
                timeout=timeout, write_array=write_array.tobytes()))
E       TypeError: 'float' object cannot be interpreted as an integer
```

It looks like this division was overlooked in the Python 2 -> Python 3 migration:
```
        number_of_samples_per_channel, _ = divmod(
            numpy_array.nbytes, (
                number_of_channels * channels_to_write.ao_resolution / 8))
```

This code is also in `in_stream.readinto()`.

--- a/generated/nidaqmx/_task_modules/in_stream.py
+++ b/generated/nidaqmx/_task_modules/in_stream.py
@@ -1190,7 +1190,7 @@ class InStream:
 
         number_of_samples_per_channel, _ = divmod(
             numpy_array.nbytes, (
-                number_of_channels * channels_to_read.ai_raw_samp_size / 8))
+                number_of_channels * channels_to_read.ai_raw_samp_size // 8))
 
         _, samples_read, _ = self._interpreter.read_raw(
             self._handle, number_of_samples_per_channel,

--- a/generated/nidaqmx/_task_modules/out_stream.py
+++ b/generated/nidaqmx/_task_modules/out_stream.py
@@ -558,8 +558,8 @@ class OutStream:
 
         number_of_samples_per_channel, _ = divmod(
             numpy_array.nbytes, (
-                number_of_channels * channels_to_write.ao_resolution / 8))
+                number_of_channels * int(channels_to_write.ao_resolution) // 8))
 
         return self._interpreter.write_raw(
-            self._handle, number_of_samples_per_channel, 
+            self._handle, number_of_samples_per_channel,
             self.auto_start, self.timeout, numpy_array)

--- a/src/codegen/templates/_task_modules/in_stream.py.mako
+++ b/src/codegen/templates/_task_modules/in_stream.py.mako
@@ -311,7 +311,7 @@ ${property_template.script_property(attribute)}\
 
         number_of_samples_per_channel, _ = divmod(
             numpy_array.nbytes, (
-                number_of_channels * channels_to_read.ai_raw_samp_size / 8))
+                number_of_channels * channels_to_read.ai_raw_samp_size // 8))
 
         _, samples_read, _ = self._interpreter.read_raw(
             self._handle, number_of_samples_per_channel,

--- a/src/codegen/templates/_task_modules/out_stream.py.mako
+++ b/src/codegen/templates/_task_modules/out_stream.py.mako
@@ -160,8 +160,8 @@ ${property_template.script_property(attribute)}\
 
         number_of_samples_per_channel, _ = divmod(
             numpy_array.nbytes, (
-                number_of_channels * channels_to_write.ao_resolution / 8))
+                number_of_channels * int(channels_to_write.ao_resolution) // 8))
 
         return self._interpreter.write_raw(
-            self._handle, number_of_samples_per_channel, 
+            self._handle, number_of_samples_per_channel,
             self.auto_start, self.timeout, numpy_array)

